nemo/collections/asr/models/rnnt_models.py

```diff
@@ -72,7 +72,10 @@ def __init__(self, cfg: DictConfig, trainer: Trainer = None):
         loss_name, loss_kwargs = self.extract_rnnt_loss_cfg(self.cfg.get("loss", None))
 
         self.loss = RNNTLoss(
-            num_classes=self.joint.num_classes_with_blank - 1, loss_name=loss_name, loss_kwargs=loss_kwargs
+            num_classes=self.joint.num_classes_with_blank - 1,
+            loss_name=loss_name,
+            loss_kwargs=loss_kwargs,
+            reduction=self.cfg.get("rnnt_reduction", "mean_batch"),
```

Comment:
probably should be in the loss config?

Reply:
This is meant to be similar to the implementation in the [CTC models](https://github.com/NVIDIA/NeMo/blob/d19146cd5cad07151f94820ff92392a29c3225f6/examples/asr/conf/conformer/conformer_ctc_bpe.yaml#L32)